src/contact.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: default.html
+pageTitle: Contact Us
+tags: nav
+order: 3
+---
+
+<h1 class="title">
+    Contact Us
+</h1>
+
+<section>
+    <h1 class="title">
+        Make an Appointment
+    </h1>
+    <h2>
+        Open Hours
+    </h2>
+    <h3>
+        Select a Date:
+    </h3>
+    <form class="calendar" action="">
+        <input type="radio" id="mon" name="date" value="monday">
+        <label for="mon">Monday</label><br>
```

Comment:
Instead of using `br` elements here, you want to control the layout and display via CSS.